docs/content/manual/upgrade/upgrade-from-traditional-network-design.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Upgrade Harvester from traditonal cluster network design (before v1.1.0)
+---
+
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
+1. Network have at least two NICs
+1. Suggest each Harvester node have at lease 6 core CPU, 16GB memory and 200GB disk
```

Comment:
Should be `node has at least`

Reply:
Appreciated for the checking. Update this typo.